python/src/iceberg/utils/binary.py

```diff
@@ -0,0 +1,25 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+"""Helper methods for working with bytes (FixedType or BinaryType) data
+"""
+
+import base64
+
+
+def base64encode(buffer: bytes) -> str:
```

Comment:
I think the utils methods added in this PR should be private methods in transforms.py instead because they are not used in Iceberg Java outside of the `transforms` package. There's no need to expose them in Python either.

Reply:
Actually, I think I'd just move this one to avoid needing to move EPOCH constants.

Reply:
Done and removed binary.py.